src/main/java/frc/robot/Robot.java

```diff
@@ -16,8 +19,9 @@
  */
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
-
   private RobotContainer m_robotContainer;
+  private IntakeSubsystem m_intakeSubsystem;
```

Comment:
It would be better to instantiate the subsystems in RobotContainer because you'll want to use them with your Command classes and button bindings